<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <!-- Action -->
    <arg name="eval_trajectory" default="false"/>

    <!-- Bag file -->
    <arg name="bag_file" default="/home/davide/Desktop/Video_to_process/voxblox_planner.bag"/>
   
    <!-- Trajectory computer -->
    <arg name="data_file_name" default="/home/davide/Desktop/drift0_01.csv"/>
    <arg name="evaluation_time_rate" default="0.025" />

<!--_______________________________ Nodes ___________________________________-->

    <!-- Bag File Player -->
    <node name="player" pkg="rosbag" type="play" output="screen" args="-r 20 -s 0 $(arg bag_file)"/>

    <!-- Trajectory computer >
    <node name="odom_difference_evaluator" pkg="active_3d_planning_app_submap_exploration" type="odom_difference.py" output="screen" if="$(arg eval_trajectory)"> 
    <param name="data_file_name" value="$(arg data_file_name)"/>
    <param name="evaluation_time_rate" value="$(arg evaluation_time_rate)" />
    </node-->

	<!--Arrow display -->
    <node name="arrow" pkg="active_3d_planning_app_submap_exploration" type="arrow.py" output="screen"/>
</launch>
